src/pages/Auth/Login/index.tsx

```diff
@@ -0,0 +1,147 @@
+import { Link } from "react-router-dom";
+
+// Components
+import Layout from "@/components/Layout";
+import { useState } from "react";
+
+const Login = () => {
+  const [forgetPassword, setForgetPassword] = useState(false);
+
+  console.log(forgetPassword, "testt");
+
+  return (
+    <Layout>
+      <div id="wrapper">
+        <div className="tf-page-title style-2">
+          <div className="container-full">
+            <div className="heading text-center">Log in</div>
+          </div>
+        </div>
+
+        <section className="flat-spacing-10">
+          <div className="container">
+            <div className="tf-grid-layout lg-col-2 tf-login-wrap">
+              <div className="tf-login-form">
+                <div
+                  id="recover"
+                  style={{ display: `${forgetPassword ? "block" : "none"}` }}>
```

Comment:
You don't need to use special quotes here.

```suggestion
                  style={{ display: forgetPassword ? "block" : "none" }}>
```